react-components/src/components/Widgets/constants.ts

```diff
@@ -0,0 +1,8 @@
+/*!
+ * Copyright 2023 Cognite AS
+ */
+
+export const X_OFFSET = 150;
+export const Y_OFFSET = 75;
+export const WIDTH_FACTOR = 0.6;
+export const HEIGHT_FACTOR = 0.8;
```

Comment:
Can we have some more explanatory names for these? E.g. `X_WINDOW_WIDGET_MARGIN` (if that's correct)?